.github/workflows/release-series-images.yml

```diff
@@ -95,33 +87,22 @@ jobs:
         # which is required for build-push-action
         IFS=,
         echo "::set-output name=taglist::${tag_list[*]}"
-
+    
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
-
-    - name: Log in to Docker Hub
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
       with:
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
-
-    - name: Log in to OSG Harbor
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
-      with:
-        registry: hub.opensciencegrid.org
-        username: ${{ secrets.OSG_HARBOR_ROBOT_USER }}
-        password: ${{ secrets.OSG_HARBOR_ROBOT_PASSWORD }}
-
-    - name: Build and push Docker image
+        driver: docker  # If not set to docker driver, it will default to docker-container
+                        # when using load for the build-push-action.
+    
+    - name: Build and push hosted-ce and osg-ce-condor images
       uses: docker/build-push-action@v2.2.2
       with:
+        load: true
         push: ${{ github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/') }}
-        context: .
+        context: ${{matrix.dockercontext}}
+        file: ${{matrix.dockerfile}}
         build-args: |
           BASE_YUM_REPO=${{ matrix.repo }}
           BASE_OSG_SERIES=${{ matrix.osg_series }}
         tags: "${{ steps.generate-tag-list.outputs.taglist }}"
-        target: ${{ matrix.image }}
-        cache-from: type=local,src=/tmp/.buildx-cache
+          
```

Comment:
Your editor's still saving files without a newline. You use Visual Studio, right? This looks maybe helpful https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files